src/SonarScanner.MSBuild.PreProcessor/PreProcessor.cs

```diff
@@ -114,25 +106,16 @@ private async Task<bool> DoExecute(ProcessedArgs localSettings)
             }
             Debug.Assert(argumentsAndRuleSets.AnalyzersSettings != null, "Not expecting the analyzers settings to be null");
 
-            if (PullRequestBaseBranch(localSettings) is { } baseBranch)
-            {
-                logger.LogInfo(Resources.MSG_Processing_PullRequest_Branch, baseBranch);
-            }
-            else
-            {
-                logger.LogDebug(Resources.MSG_Processing_PullRequest_NoBranch);
-            }
-
             using var cache = new CacheProcessor(server, localSettings, buildSettings, logger);
             await cache.Execute();
 
             var version = await server.GetServerVersion();
-            var additionalSettings = new Dictionary<string, string> {
+            var additionalSettings = new Dictionary<string, string>
+            {
                 { nameof(cache.UnchangedFilesPath), cache.UnchangedFilesPath },
```

Comment:
Just to confirm that I understood right. We add this here in order to be serialized and available for analyzers to read.

Reply:
Exactly.